File: src/main/java/itmo/is/project/mapper/module/ProductionModuleBlueprintMapper.java
package itmo.is.project.mapper.module;

import itmo.is.project.dto.module.production.ProductionModuleBlueprintDto;
import itmo.is.project.mapper.EntityMapper;
import itmo.is.project.model.module.production.ProductionModuleBlueprint;
import org.mapstruct.Mapper;

@Mapper(componentModel = "spring")
public interface ProductionModuleBlueprintMapper
        extends EntityMapper<ProductionModuleBlueprintDto, ProductionModuleBlueprint> {
}

File: src/main/kotlin/itmo/isproject/mapper/module/production/ProductionModuleBlueprintMapper.kt
package itmo.isproject.mapper.module.production

import itmo.isproject.dto.module.production.ProductionModuleBlueprintDto
import itmo.isproject.mapper.EntityMapper
import itmo.isproject.model.module.production.ProductionModuleBlueprint
import org.mapstruct.Mapper

@Mapper
interface ProductionModuleBlueprintMapper : EntityMapper<ProductionModuleBlueprintDto, ProductionModuleBlueprint>
